app/components/provider_interface/course_details_component.rb

```diff
@@ -6,16 +6,17 @@ class CourseDetailsComponent < ViewComponent::Base
     FUNDING_TYPES = { apprenticeship: :apprenticeship, salary: :salaried, fee: :fee_paying }.freeze
 
     attr_reader :application_choice, :provider_name_and_code, :course_name_and_code,
-                :cycle, :preferred_location, :study_mode, :qualification
```

Comment:
instead of amending the current course details component, can we add a new component and repurpose this to be read only? So we show the new component if the provider has make decision access and the feature flag is on, and the old one (this one) if the feature flag is off and the provider has no make decision access? This will avoid having the feature flag everywhere, also will guard against providers with no make decision access